packs/equipment/arboreals-revenge.json

```diff
@@ -21,7 +21,7 @@
e eyes.</p>\n<p><em>Arboreal's revenge</em> is a haunted firearm constructed with wood hewn from a living arboreal, slaying the arboreal and trapping its spirit in the firearm. Perhaps a patient wielder could one day put the arboreal's spirit to rest, or at least come to terms with it, abating the drawbacks of the weapon while keeping its advantages.</p>\n<p>When you first fire <em>arboreal's revenge</em>, the blunderbuss fuses with you, after which it's almost impossible to remove it from your possession with a @UUID[Compendium.pf2e.spells-srd.Item.Cleanse Affliction] or similar effect, like many cursed items. You gain weakness 5 to fire and the flat check for you to recover from persistent fire damage increases from 15 to 17, or from 10 to 12 if you receive appropriate help.</p>\n<p>Whenever you wield <em>arboreal's revenge</em>, the arboreal spirit bound to the blunderbuss attempts to overtake you. The wood handgrip sprouts roots which grow around your hand, binding both of your hands to the firearm. You gain a +4 circumstance bonus to your Reflex DC when defending against checks to @UUID[Compendium.pf2e.actionspf2e.Item.Disarm] you. However, you can't Release your grip on the blunderbuss until you forcibly tear off the roots, though they do allow you to move your hands enough (and only enough) to reload and fire the blunderbuss. Tearing off the roots takes 1 minute.</p>\n<p>The arboreal haunting the firearm can use the following two effects whenever it wants, often using them at random. You can coax it into performing them each intentionally once per day, with the following activations.</p><hr /><p><strong>Activate</strong> <span class=\"action-glyph\">f</span> command</p>\n<p><strong>Frequency</strong> once per day</p><hr /><p><strong>Effect</strong> The arboreal haunting the blunderbuss attempts to regrow itself into a new physical body by sprouting from the bodies of all the creatures that the firearm recently harmed. You and all creatures within 60 feet damaged by <em>arboreal's revenge</em> within the last minute must attempt a @Check[fortitude|dc:20|name:arboreal's revenge] saving throw. On a failure, they take a -10-foot circumstance penalty to their Speeds as roots rapidly sprout from their wounds, which lasts until they @UUID[Compendium.pf2e.actionspf2e.Item.Escape] (DC 20).</p>\n<p>@UUID[Compendium.pf2e.equipment-effects.Item.Effect: Arboreal's Revenge (Speed Penalty)]</p><hr /><p><strong>Activate</strong> <span class=\"action-glyph\">2</span> command, Interact</p>\n<p><strong>Frequency</strong> once per day</p><hr /><p><strong>Effect</strong> You cast @UUID[Compendium.pf2e.spells-srd.Item.Oaken Resilience] from the blunderbuss as a 2nd- level primal spell. However, the weakness to fire imposed by <em>oaken resilience</em> is cumulative with the weakness to fire imposed by this weapon's curse, for a total of weakness 8 to fire.</p>\n<p>@UUID[Compendium.pf2e.spell-effects.Item.Spell Effect: Oaken Resilience (Arboreal's Revenge)]</p>"
```

Comment:
This was from the extract - "{Spell Effect: Oaken Resilience (Arboreal's Revenge)}" was removed.